source/uwp/Renderer/idl/AdaptiveCards.Rendering.Uwp.idl

```diff
@@ -530,15 +530,17 @@ AdaptiveNamespaceStart
         Content
     } FallbackType;
 
-    [version(NTDDI_WIN10_RS1)]
+    [version(NTDDI_WIN10_RS1), flags]
 #ifdef ADAPTIVE_CARDS_WINDOWS
     [contract(InternalContract, 1)][internal]
 #endif
     typedef[v1_enum] enum BleedDirection {
```

Comment:
>v1_enum] [](start = 12, length = 8)

Curious: What is our versioning story in general and why do we even have this explicit typedef in place? Have we looked at the WinRT API design guidelines or gone through an API review at all all with Wiswal and Co?

Reply:
I believe windows API review is scoped for APIs that ship out of windows (which this does not) and thus this hasn't been formally reviewed. We did get some informal guidance from them on some things early on when we were designing the API.

---
In reply to: [279813721](https://github.com/Microsoft/AdaptiveCards/pull/2802#discussion_r279813721) [](ancestors = 279813721)